### PR TITLE
Revert "RHPAM-2721 :Clone project via http can not be re-enabled (#938)"

### PR DESCRIPTION
**Pull request by jomarko**
This reverts commit dec68e71722069e248f2c4dd1b5f7ed3c7d490f9.

Caused issues to gwt showcase webapps

### CODE REVIEW

**Comment by jomarko** [comment]
jenkins execute full downstream build

File: uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemProvider.java [hunk 1]
```diff
@@ -49,6 +49,38 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
+import java.io.File;
+import java.io.FileNotFoundException;
```
Comment (yesamer) [inline uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemProvider.java:53]: @jomarko These imports are dupliated: please refer lines 19-50

File: uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemProvider.java [hunk 2]
```diff
@@ -65,6 +97,13 @@
 import org.eclipse.jgit.transport.CredentialsProvider;
 import org.eclipse.jgit.transport.ReceiveCommand;
 import org.eclipse.jgit.transport.ReceivePack;
+import org.eclipse.jgit.transport.ServiceMayNotContinueException;
+import org.eclipse.jgit.transport.SshSessionFactory;
+import org.eclipse.jgit.transport.UploadPack;
+import org.eclipse.jgit.transport.UsernamePasswordCredentialsProvider;
+import org.eclipse.jgit.transport.CredentialsProvider;
+import org.eclipse.jgit.transport.ReceiveCommand;
+import org.eclipse.jgit.transport.ReceivePack;
```
Comment (yesamer) [inline uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemProvider.java:106]: @jomarko Duplicated import here too

File: uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemProvider.java [hunk 3]
```diff
@@ -127,15 +166,27 @@
...
 import org.uberfire.java.nio.file.StandardOpenOption;
 import org.uberfire.java.nio.file.WatchEvent;
+import org.uberfire.java.nio.file.FileSystemAlreadyExistsException;
+import org.uberfire.java.nio.file.FileSystemNotFoundException;
+import org.uberfire.java.nio.file.LinkOption;
+import org.uberfire.java.nio.file.NoSuchFileException;
+import org.uberfire.java.nio.file.NotDirectoryException;
+import org.uberfire.java.nio.file.OpenOption;
+import org.uberfire.java.nio.file.Option;
+import org.uberfire.java.nio.file.Path;
```
Comment (yesamer) [inline uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemProvider.java:176]: @jomarko Duplication

**Comment by RishiRajAnand** [comment]
@jomarko @ederign @mareknovotny I can confirm that the errors in gwt web apps is due to the [PR](https://github.com/kiegroup/appformer/commit/dec68e71722069e248f2c4dd1b5f7ed3c7d490f9). I am working on the fix. In the meantime we can merge this revert PR, since it is a blocker.  
Thanks

File: uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemProvider.java (hunk 3 above)
Comment (RishiRajAnand) [reply uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemProvider.java:176]: @yesamer thanks for pointing these duplication issues, I'll make sure to include them as part of permanent fix.

**Pull request merged**